assets/css/popup.css

```diff
@@ -0,0 +1,151 @@
+.container .img-pop-up-button {
```

Comment:
On top of this file, add a small documentation that explains:
* Where this CSS will be used.
* Overall behavior.